src/Crowbar.cpp

```diff
@@ -4,8 +4,9 @@
 #include <zframe.h>
 
 #include "Crowbar.h"
-#include "boost/thread.hpp"
-#include "g2log.hpp"
+#include <boost/thread.hpp>
+#include <g3log/g3log.hpp>
+#include <Death.h>
```

Comment:
Thanks. I will go through what John did and make sure I don't overwrite his changes